src/schema/v2/phoneNumber.ts

```diff
@@ -108,12 +108,36 @@ const PhoneNumberType: GraphQLObjectType<
       args: {
         format: PhoneNumberFormats,
       },
+      description: "A formatted phone number. Null if isValid is false",
       resolve: ({ parsedPhone, phoneUtil }, { format }) =>
-        parsedPhone && phoneUtil.format(parsedPhone, format),
+        parsedPhone && phoneUtil.isValidNumber(parsedPhone)
+          ? phoneUtil.format(parsedPhone, format)
+          : null,
```

Comment:
Is this a behavioral change?

Reply:
youre right. we don't need this change. an invalid number can still be parsed and show while being invalid:
![image](https://github.com/user-attachments/assets/f2652332-7c8d-4a1e-95b5-a1a14c21f810)
![image](https://github.com/user-attachments/assets/740b6c6a-6bee-4c6c-9f4d-6c49db7468e7)

I'll update this.